spec/requests/create_work_spec.rb

```diff
@@ -645,9 +646,9 @@
             license: "CC0-1.0",
             upload_type: "browser",
             release: "embargo",
-            "embargo(1i)": "2030",
-            "embargo(2i)": "09",
-            "embargo(3i)": "01"
+            "embargo_date(1i)": "2030",
+            "embargo_date(2i)": "09",
+            "embargo_date(3i)": "01"
```

Comment:
These have been wrong ever since they were added. Typos, prolly. And now they're permitted.